src/mesh/Draco_Mesh.cc

```diff
@@ -433,6 +440,220 @@ Draco_Mesh::compute_node_vec_indx_map(const std::vector<unsigned> &indx_type,
   return nodes_to_indx_map;
 }
 
+//------------------------------------------------------------------------------------------------//
+/*!
+ * \brief Build a cell-to-cell linkage across corners
+ *
+ */
```

Comment:
Please add `\param` for this function.

Reply:
Thank you for catching this, these comment fields have been added.